packages/e2e-test-utils/src/visitWidgetScreen.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import visitAdminPage from './visitAdminPage';
+
+async function visitWidgetScreen() {
```

Comment:
Maybe add a comment or rename so it's clear that it's for the block-based widgets screen?

We could also make it work for the classic widgets screen by making the `page.evaluate` calls below more robust (i.e. check if `wp.data` is defined first)

Reply:
Name and code based on https://github.com/WordPress/gutenberg/blob/4f5360e47c9874e52fd74a6b7693e90e107fa2d2/packages/e2e-tests/specs/widgets/editing-widgets.test.js#L23-L45

Reply:
Gutenberg only tests the block-based widgets screen, but we have to test the old one as well. So it makes sense to update the function accordingly so it can work for both.

If that's not feasible, adding some explanatory docblock would be helpful.

Reply:
I have changed the name.